packages/@okta/vuepress-site/docs/reference/api/templates/index.md

```diff
@@ -371,13 +371,13 @@ All templates have the following properties:
 | template               | Text of the template, including any [macros](#sms-template-macros). | String (See note below)                                        | FALSE    | 1         | 161       |
 | created                | Timestamp when template was created                                 | String (ISO-8601)                                              | TRUE     | N/A       | N/A       |
 | lastUpdated            | Timestamp when template was last updated                            | String (ISO-8601)                                              | TRUE     | N/A       | N/A       |
-| translations           | Array of [translations](#translation-attributes)                    | Array                                                          | N/A      | N/A       | N/A       |
+| translations           | A list of [translations](#translation-attributes)                    | Object                                                          | N/A      | N/A       | N/A       |
```

Comment:
Not sure this terminology is consistent with the rest of our API docs. See this [section on sign-on policy](https://developer.okta.com/docs/reference/api/policy/#signon-action-object) for an example of where I'm coming from.

- `List` is confusing to me as it implies an array, not a dictionary or key-value map. Perhaps something like `A key-value map for language translations based on a [user's language preferences](link-to-some-explanation-of-language-preferences)`
- The `Object` data type should be a `Translation Attributes` data type with a link to the section below.

Reply:
I will update the description, but the type should be one of the [JSON Data Types](https://www.w3schools.com/js/js_json_datatypes.asp).